pkg/gitnessclient/gitness_types.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package gitnessclient
+
+import "time"
+
+type apiMembershipResponse struct {
```

Comment:
Is there a specific reason for the "api" prefix in the types inside this file? `apiMembershipResponse`, `apiSpace` etc.
If there's no naming conflicts I'd remove the prefix.